@xen-orchestra/lite/src/libs/utils.ts

```diff
@@ -33,6 +34,11 @@ const iconsByType = {
   boolean: faSquareCheck,
   enum: faList,
 };
+export function formatSize(bytes: number) {
+  return bytes != null
```

Comment:
If `bytes` could be `null` or `undefined`, maybe we should change its type in function signature?

Reply:
We should also add a basic typing in `human-format.d.ts`, something like this:

```ts
declare module "human-format" {
  function bytes(value: number): string;
  function humanFormat(
    value: number,
    options: {
      maxDecimals?: number;
      decimals?: number;
      scale?: string;
      unit?: string;
      separator?: string;
      prefix?: string;
    }
  ): string;
  export = humanFormat;
}
```

Reply:
For the type declaration of `human-format`, this has already been done in another [PR](https://github.com/vatesfr/xen-orchestra/pull/6421/files#diff-10f0d03398f1aefe165d4bec5045863e3bd966cd4509c7267a7fd6c6b67a5420R2)